runtime/crab/src/pallets/governance/origin.rs

```diff
@@ -77,5 +83,44 @@ pub mod custom_origins {
 		() => {}
 	}
 	decl_unit_ensures!(ReferendumCanceller, ReferendumKiller, WhitelistedCaller, GeneralAdmin);
```

Comment:
Why the `decl_unit_ensures` in Crab doesn't contain the `MediumSpender` and `BigSpender`?